astroid/nodes/__init__.py

```diff
@@ -306,4 +307,5 @@
     "With",
     "Yield",
     "YieldFrom",
+    "Position",
```

Comment:
I think, we tried to keep `__all__` sorted. I.e. it should probably be between `Pass` and `Raise`.

Reply:
I'll change it!